src/Engine/Messages/MessageModel.cs

```diff
@@ -52,6 +52,12 @@ public class MessageModel : ISerializable
         }
 
         [DataMember]
+        public string ID {
```

Comment:
Well, I've tested this new frontend version with my changes, against the 1.0 smuxi cloud, and obviously it doesn't get links, but it works (no crashes).